scripts/install_stock.sh

```diff
@@ -36,6 +36,10 @@ wget --continue --quiet https://github.com/opencontainers/runc/releases/download
 mv runc.amd64 runc
 sudo install -D -m0755 runc /usr/local/sbin/runc
 
+wget --continue --quiet https://storage.googleapis.com/gvisor/releases/release/latest/x86_64/runsc 
```

Comment:
let's fix the release to a concrete version that is currently the latest stable one. this would allow to avoid incompatibility problems that might come with new versions.

Reply:
Changed runsc release to a fixed one.